tern/formats/default/generator.py

```diff
@@ -113,8 +114,14 @@ def get_layer_info_list(layer):
         file_level_licenses = ", ".join(layer_file_licenses_list)
 
     for package in layer.packages:
+        pkg_licenses = package.pkg_license
+        if package.pkg_format == 'deb':
+            pkg_licenses = wrap(", ".join(package.pkg_licenses)) or ['']
         package_list.add_row([package.name, package.version,
-                              package.pkg_license, package.pkg_format])
+                              pkg_licenses[0], package.pkg_format])
+
+        for pkg_license in pkg_licenses[1:]:
```

Comment:
Can you add a comment just above on line 22 describing this extra loop? Something like:
```
# Each wrapped line must be added as a new row under the same package
```